src/components/pages/about-us/impressions.tsx

```diff
@@ -59,7 +59,7 @@ export const Impressions = ({ impressions }: ImpressionsProps) => {
         kicker={t<string>('about-us.impressions.title')}
       >
         <MobileOnlyText>{t('about-us.description')}</MobileOnlyText>
-        {t('about-us.impressions.text')}
+        {tWithParagraphs('about-us.impressions.text')}
```

Comment:
That's cool! I wonder if there are some middleware available in the transition engine to always convert newlines to paragraphs. I like the approach but I would prefer to play it more conservative:

+ Put it where we need it (like you started it) and do not sprinkle `tWithParagraphs` everywhere
+ If we want it everywhere, find a way to be not so invasive. Either a middleware or some other processing hook if possible. Last ressort is of course the customized `t` function as here.

Reply:
Yes my plan is also to use the new feature only when needed. Currently, according to the design, there are only two places anyway. I have added the function to the other place as well.